website/docs/reference/dbt-jinja-functions/return.md

```diff
@@ -9,7 +9,12 @@ __Args__:
 
  * `data`: The data to return to the caller
 
-The `return` function can be used in macros to return data to the caller. The type of the data (dict, list, int, etc) will be preserved through the `return` call.
+The `return` function can be used in macros to return data to the caller. The type of the data (dict, list, int, etc) will be preserved through the `return` call.  You can use the `return` function in the following ways within your macros: as an expression or as a statement.
```

Comment:
@dbeatty10 added this here as i think it'd be helpful. wdyt?

Reply:
Thanks for doing that! Looks great to me 👍